test/jdk/com/sun/management/ThreadMXBean/ThreadAllocatedMemory.java

```diff
@@ -152,7 +158,7 @@ private static void testGetThreadAllocatedBytes()
         try {
             curThread.join();
         } catch (InterruptedException e) {
```

Comment:
should it just let `InterruptedException` be thrown rather than swallowing it?  This pattern appears multiple places in this test and ThreadAllocatedMemoryArray.java test.

Reply:
The original implementation grabbed this code from, e.g., test/jdk/java/lang/management/ThreadMXBean/ThreadUserTime.java. In the latter, InterruptedException causes the test to fail, but for some reason now lost, ThreadAllocatedMemory.java doesn't do that, but ThreadAllocatedMemoryArray.java does.

Reply:
Since you are on this file, it may be good to clean them up.

Reply:
Done.